pkgs/development/tools/build-managers/mill/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, lib, fetchurl, makeWrapper, jre, gnused }:
+ stdenv.mkDerivation rec {
+  name = "mill-${version}";
+  version = "0.3.5";
+    src = fetchurl {
+     url = "https://github.com/lihaoyi/mill/releases/download/${version}/${version}";
+     sha256 = "19ka81f6vjr85gd8cadn0fv0i0qcdspx2skslfksklxdxs2gasf8";
+  };
+   propagatedBuildInputs = [ jre ] ;
+  buildInputs = [ makeWrapper gnused ] ;
+   phases = "installPhase";
```

Comment:
What's this line for?

Reply:
It is used to override all other phases with my own defined `installPhase`. This will crash on the unpack phase for example, since it is a binary. Maybe there are better ways doing this? I think I took this from the nix pills.